src/Cli/dotnet/Commands/Reference/List/ReferenceListCommand.cs

```diff
@@ -44,7 +44,7 @@ public override int Execute()
         Reporter.Output.WriteLine(new string('-', CliStrings.ProjectReferenceOneOrMore.Length));
         foreach (var p2p in p2ps)
         {
-            Reporter.Output.WriteLine(p2p.Include);
+            Reporter.Output.WriteLine(p2p.EvaluatedInclude);
```

Comment:
I don't think the Project is really evaluated at this point; it looks like it just loaded the XML. You might have to call msbuildProj.GetEvaluatedProject() to get a real Project, from which you can call GetItems("ProjectReference") and iterate through that instead.